extensions/amp-analytics/0.1/events.js

```diff
@@ -147,13 +148,45 @@ export class CustomEventTracker extends EventTracker {
     const targetReady =
         this.root.getElement(context, selector, selectionMethod);
 
-    let observers = this.observers_[eventType];
-    if (!observers) {
-      observers = new Observable();
-      this.observers_[eventType] = observers;
+    const isSandboxEvent = startsWith(eventType, 'sandbox-');
+
+    // Push recent events if any.
+    let buffer;
```

Comment:
Ah, Sorry, you mean only change it in `#add()`. Sure I can do that since I only need to read buffer here.